bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -46,20 +46,22 @@
   {% if show_mr2_promo %}
     {% include 'mozorg/home/includes/mr2-promo.html' %}
   {% else %}
-    {% call hero(
-    class='mzp-c-hero mzp-has-image',
-    include_cta=True,
-    image_url='img/home/laptop.png'
+    {% call split(
+      image_url='img/home/laptop.png',
+      include_highres_image=True,
+      block_class='c-home-hero mzp-l-split-center-on-sm-md mzp-t-split-nospace',
+      body_class=None,
+      media_class='mzp-l-split-media-overflow',
```

Comment:
**Question/suggestion (non-blocking):** Curious why we're placing an `overflow` class here? I think it looks better without it and would closely resemble our main EN/FR/DE homepages?

Reply:
Only because it replicates the behavior of Hero, but I agree it's probably better without it. I'll update.